Mediapipe.Net/Calculators/ICalculator.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) homuler and Vignette
+// This file is part of MediaPipe.NET.
+// MediaPipe.NET is licensed under the MIT License. See LICENSE for details.
+
+using System;
+
+namespace Mediapipe.Net.Calculators;
+
+public interface ICalculator<U, T> : IDisposable
+{
+    public void Run();
+    public U Perform(U frame, out T result);
```

Comment:
I think `StartRun()` here is a better name. It's more explicit and reflects on the MediaPipe equivalent.
Also, `Perform()` makes me think of art or something lol. Maybe `Process`?
Maybe even better: use a `Send()` function to send in image frames and then `Receive` to receive from any output stream poller.

Reply:
`Perform` is a blocking call where we send and then receive